src/plugins/intel_gpu/src/plugin/ops/istft.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/istft.hpp"
+
+#include "intel_gpu/plugin/common_utils.hpp"
+#include "intel_gpu/plugin/program_builder.hpp"
+#include "intel_gpu/primitives/istft.hpp"
+
+namespace ov::intel_gpu {
+
+static void CreateISTFTOp(ProgramBuilder& p, const std::shared_ptr<ov::op::v16::ISTFT>& op) {
+    validate_inputs_count(op, {4, 5});
+    auto inputs = p.GetInputInfo(op);
+    if (inputs.size() == 4) {
+        auto prim = cldnn::ISTFT(layer_type_name_ID(op), inputs[0], inputs[1], inputs[2], inputs[3], op->get_center(), op->get_normalized());
+        p.add_primitive(*op, prim);
+    } else {
+        auto prim = cldnn::ISTFT(layer_type_name_ID(op), inputs[0], inputs[1], inputs[2], inputs[3], inputs[4], op->get_center(), op->get_normalized());
+        p.add_primitive(*op, prim);
+    }
```

Comment:
I think it's not actually needed to enumerate all the inputs, we can just slightly update ISTFT like this:
```
    ISTFT(const primitive_id& id,
          const std::vector<input_info>& inputs,
          const bool center,
          const bool normalized)
        : primitive_base(id, inputs),
          center(center),
          normalized(normalized) {}
```
And simplify primitive creation:
```suggestion
    auto prim = cldnn::ISTFT(layer_type_name_ID(op), inputs, op->get_center(), op->get_normalized());
    p.add_primitive(*op, prim);
```

Reply:
Fixed.